python/cudf/cudf/core/column/string.py

```diff
@@ -4986,7 +4986,7 @@ def character_ngrams(
         return result
 
     def hash_character_ngrams(
-        self, n: int = 5, as_list: bool = False
+        self, n: int = 5, seed: np.uint32 = 0, as_list: bool = False
```

Comment:
Could you put `seed` as the last argument here?

Technically this would be a API breaking for user previously passing `n` and `as_list` by position e.g. `.str.hash_character_ngrams(2, True)`